.github/workflows/update-repo.yml

```
@@ -0,0 +1,44 @@
+name: Update plugin repo
+on:
+  schedule:
+    - cron: "0 */6 * * *"
```

Currently, my script runs every 30 minutes.
If we want to keep the same schedule:

```suggestion
    - cron: "*/30 * * * *"
```

I am no saying we have to run it every 30 minutes. 

That being said, we should mention somewhere (web site, README) how often the plugin data is updated.